src/styles/variables.scss

```diff
@@ -0,0 +1,11 @@
+$stopwatch-size: 80vmin;
+$stopwatch-seconds-height: 38vmin;
+$stopwatch-seconds-width: 1.5vmin;
+$stopwatch-seconds-color: #2c8000;
+$stopwatch-minutes-height: 20vmin;
+$stopwatch-minutes-width: 3vmin;
+$stopwatch-minutes-color: #0700ff;
+$stopwatch-circle-size: 5vmin;
+$stopwatch-circle-color: #f6a603;
+$minute-duration: 60s;
+$hour-duration: $minute-duration * 60;
```

Comment:
Using arithmetic operations directly in the value assignment can lead to confusion and reduce maintainability. Consider defining a variable for the number of minutes in an hour to improve readability, for example: `$minutes-in-hour: 60;` and then `$hour-duration: $minute-duration * $minutes-in-hour;`.